src/storage/config.rs

```diff
@@ -307,9 +310,17 @@ impl Default for IORateLimitConfig {
 }
 
 impl IORateLimitConfig {
-    pub fn build(&self, enable_statistics: bool) -> IORateLimiter {
-        let limiter = IORateLimiter::new(self.mode, self.strict, enable_statistics);
-        limiter.set_io_rate_limit(self.max_bytes_per_sec.0 as usize);
+    pub fn build(&mut self, stats_collector_enabled: bool) -> IORateLimiter {
```

Comment:
what's the overhead of stats_collector_enabled being true? It looks that the IO statistics collector would lead to lock contention for IO path. 
Wondering the typical workload like sysbench/tpc-c's performance diff with this flag true and false.

Reply:
If you are referring to the lock in limiter, it won't be accessed if IO limit is not reached.